### ISSUE TITLE
Github project desciption says Vaadin 14 

### ISSUE THREAD

**Issue by geertschuring** [vaadin/vaadin-form-example#223]
The github projects description says it uses Vaandin 14 while it has been updated to Vaadin 24 if im not mistaken.

**Comment by samie** [comment]
Yes, there are separate branches for the different versions [V24](https://github.com/vaadin/vaadin-form-example/tree/v24) and [V14](https://github.com/vaadin/vaadin-form-example/tree/v14). The[ deployed version](https://vaadin-form-example.demo.vaadin.com/) is still running V14.
